docs/GettingStarted.md

```diff
@@ -2068,21 +2068,11 @@ Datadog.configure { |c| c.diagnostics.startup_logs.enabled = true }
 
 By default, this will be activated whenever `ddtrace` detects the application is running in a non-development environment.
 
-### Sampling
```

Comment:
I think we should keep this heading, as the two next sections (application-side sampling, priority sampling) are supposed to be under it.

Reply:
Added back 👍